.vscode/launch.json

```diff
@@ -40,6 +40,16 @@
         "${workspaceFolder}/out/test/**/*.js"
       ],
       "preLaunchTask": "npm: watch"
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug Gulp task",
+      "program": "${workspaceFolder}/node_modules/gulp/bin/gulp.js",
+      "args": [
+        // "bump", "--type", "minor"
+        "bump", "--specific", "99.99.0-appv.127"
```

Comment:
Should this be here?

Reply:
I thought to include this, as it was a way to debug gulp tasks using vscode. If you don't want the example in there, I can take it out

Reply:
Ahhh Ok.  I was a confused because of the very specific version number instead of asking the user.

Reply:
Commented out the specific lines and added a short comment explaining how to use.